<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tc_Info</class>
 <widget class="QWidget" name="Tc_Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>690</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(215, 233, 237);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="title_widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_35">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="hide_btn">
        <property name="text">
         <string>——</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="info_widget" native="true">
     <widget class="QWidget" name="info_title_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="1">
        <widget class="QLabel" name="tc_warning_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 74, 74);</string>
         </property>
         <property name="text">
          <string>完善后才可进行其他操作</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_29">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_30">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="head_label">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>       头像</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_31">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_32">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="title_label">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(138, 167, 175);</string>
         </property>
         <property name="text">
          <string>个人信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="info_content_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>721</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(138, 167, 175);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="2">
        <widget class="QLineEdit" name="tc_sex_tx">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent;
border-color:#a4ccd4;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="tc_email_tx">
         <property name="styleSheet">
          <string notr="true">background:transparent;
border-color:#a4ccd4;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_33">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="tc_phnum">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>电话</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="tc_dep">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>院系</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="tc_academy_tx">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent;
border-color:#a4ccd4;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="tc_email">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>邮箱</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="tc_tell_tx">
         <property name="styleSheet">
          <string notr="true">background:transparent;
border-color:#a4ccd4;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="tc_name">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_34">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="tc_name_tx">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent;
border-color:#a4ccd4;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="tc_qq_tx">
         <property name="styleSheet">
          <string notr="true">background:transparent;
border-color:#a4ccd4;
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="tc_qq">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>QQ</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="tc_sex">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>性别</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="btn_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>721</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_37">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_36">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="return_btn">
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_38">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
